File: crates/dashboard-model/src/victron_controller/dashboard/from_0_1_0_command_set_debug_full_charge.rs


pub fn convert__command__set_debug_full_charge__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::command::SetDebugFullCharge) -> crate::victron_controller::dashboard::command::SetDebugFullCharge {
    crate::victron_controller::dashboard::command::SetDebugFullCharge {
        value: serde_json::from_value(serde_json::to_value(&from.value).unwrap()).unwrap(),
    }
}

File: web/src/model/victron_controller/dashboard/v0_2_0/from_0_1_0_command_set-debug-full-charge.ts
// @ts-nocheck
import {SetDebugFullCharge as v0_1_0_SetDebugFullCharge} from '../v0_1_0/Command'
import {SetDebugFullCharge as v0_2_0_SetDebugFullCharge} from './Command'

export function convert__command__set_debug_full_charge__from__0_1_0(from: v0_1_0_SetDebugFullCharge): v0_2_0_SetDebugFullCharge {
    return new v0_2_0_SetDebugFullCharge (
        JSON.parse(JSON.stringify(from.value))
    )
}